docs/index.rst

```diff
@@ -79,54 +83,50 @@ Overview
 Features
 --------
 
-- A rich set of data visualizations
-- An easy-to-use interface for exploring and visualizing data
-- Create and share dashboards
-- Enterprise-ready authentication with integration with major authentication
-  providers (database, OpenID, LDAP, OAuth & REMOTE_USER through
-  Flask AppBuilder)
-- An extensible, high-granularity security/permission model allowing
-  intricate rules on who can access individual features and the dataset
-- A simple semantic layer, allowing users to control how data sources are
-  displayed in the UI by defining which fields should show up in which
-  drop-down and which aggregation and function metrics are made available
-  to the user
-- Integration with most SQL-speaking RDBMS through SQLAlchemy
-- Deep integration with Druid.io
+- An intuitive interface to explore and visualize datasets, and create interactive dashboards.
+- A wide array of beautiful visualizations to showcase your data.
+- Easy, code-free, user flows to drill down and slice and dice the data underlying exposed dashboards. The dashboards and charts act as a starting point for deeper analysis.
+- A state of the art SQL editor/IDE exposing a rich metadata browser, and an easy workflow to create visualizations out of any result set.
+- An extensible, high granularity security model allowing intricate rules on who can access which product features and datasets. Integration with major authentication backends (database, OpenID, LDAP, OAuth, REMOTE_USER, ...)
+- A lightweight semantic layer, allowing to control how data sources are exposed to the user by defining dimensions and metrics
+- Out of the box support for most SQL-speaking databases
+- Deep integration with Druid allows for Superset to stay blazing fast while slicing and dicing large, realtime datasets
+- Fast loading dashboards with configurable caching
+
 
 Databases
 ---------
 
 The following RDBMS are currently supported:
 
-- `Amazon Athena <https://aws.amazon.com/athena/>`_
+- `Amazon Athena <https://superset.incubator.apache.org/installation.html#aws-athena>`_
```

Comment:
if this is linking to a page in the docs, do we need the full url? I'd guess `/installation.html#aws-athena` would be sufficient

Reply:
I just tried and it doesn't work, probably because it's .rst so the whole site is generated differently and doesn't work the same way? (I'm no .rst expert...)

Reply:
I validated that the following change works for me when building the html and running locally, I think this should be safe to do:
```diff
diff --git a/docs/index.rst b/docs/index.rst
index c1af917c1..42d9c28a2 100644
--- a/docs/index.rst
+++ b/docs/index.rst
@@ -99,7 +99,7 @@ Databases
 
 The following RDBMS are currently supported:
 
-- `Amazon Athena <https://superset.incubator.apache.org/installation.html#aws-athena>`_
+- `Amazon Athena </installation.html#aws-athena>`_
 - `Amazon Redshift <https://aws.amazon.com/redshift/>`_
 - `Apache Drill <https://superset.incubator.apache.org/installation.html#apache-drill>`_
 - `Apache Druid <https://superset.incubator.apache.org/installation.html#druid>`_
```

Reply:
Awesome, thanks for validating. Updated all the links accordingly.